### ISSUE TITLE
fail to run the exe 

### ISSUE THREAD

**Issue by ty-hei** [marccasian/KaryML-Framework#2]
My problem is that after installing all the requirements, I still cannot make the exe file run correctly. Who can give me some advice?

**Comment by marccasian** [comment]
### What it was built on?
- Operating system: Windows 10
- System architecture: 64 bit
- Mainly `Python 3.8` and for the classifiers `Java 11 (JDK version 11.0.7)`; out of this Java code we generated the exe as the code is not public. (Part of another work prof. Czibula Gabriela research group works on)

### The version number of the python package it depends on? 
- You can run `py -3 -m pip install -r requirements.txt` to install required python dependencies.
Unfortunately, I don't have the snapshot of the specific versions. Although if you can send me more specific errors (stack traces) you face I might be able to guide you to fix them & update the Github readme.md file 

### What should I do to run this program correctly?
There is a `.bat` script used to run (under Windows) the UI of the app. https://github.com/marccasian/KaryML-Framework/blob/master/StartKaryML-GUI.bat this will give you a Visual minimalistic app you can use to run the framework. 
Also the framework can be run by executing `py -3 ./GUI/KaryMl_UI.py.` If you want to run just the exe, here are the details:

CMD Line: `F-SOMv1.exe -features FEATURES_FILE_PATH -out CLASSIFIER_OUTPUT_PATH [-nofuzzy] [-weights FEATURE_WEIGHTS_FILE_PATH] [-lr LEARNING_RATE] [-e EPOCHS] [-r ROWS] [-c COLS] [-neurons INITIAL_NEURONS_FILE_PATH] [-deserialize] [-chpairs NUMBER_OF_CHROMOSOME_PAIRS] [-aqe AQE_OUT_FILE_PATH] [-manhattan]`
- `-features` FEATURES FILE PATH` path to the file containing the features used during classifier training
- `-out CLASSIFIER OUTPUT PATH` path to the file where the classifier will dump the resulting self organized map of neurons
- `-nofuzzy` if present, fuzzy version of SOM will be used as classifier [CCMI16], otherwise classic SOM version
- `-weights FEATURE WEIGHTS FILE PATH` path to the file containing the weights assigned to each used feature. If present, the used distance function will be Weighted Euclidean Distance, otherwise Euclidean distance will be used
- `-lr LEARNING RATE` the value for the initial learning rate used during classifier training, default value is 0.1
- `-e EPOCHS` number of training epochs, default value is 200000
- `-r ROWS` number of rows for the map to train, default value is 25
- `-c COLS` number of columns for the map to train, default value is 25
- `-neurons INITIAL NEURONS FILE PATH` path to the file containing serialized initial neurons map state or where to serialize the initial neurons map state, depending on the presence of -deserialize. If the path does not exists than and -deserialize is missing, than the initial neurons map state will be serialized in a file having the following format: dd-MM-yyyy-HH-mm-ss.neurons
- `-deserialize` if present and given path for serialized initial map state exists, than the given neurons state will be used as initial map state, otherwise a random state will be generated
- `-chpairs NUMBER OF CHROMOSOME PAIRS` the number of clusters to be used by the fuzzy classifier, default value is 23
- `-aqe AQE OUT FILE PATH` path to a file where will be saved aqe values among the training epochs. AQE will be computed each 100 epochs
-manhattan if present used distance function will be Manhattan Distance
Please let me know if there is anything you'll need. If needed, we can look into scheduling a call to discuss further.

**Comment by ty-hei** [comment]
Thank you for help, It appears more clearly！

**Comment by pig12138** [comment]
Hello, now I have encountered such a problem.
After I run py-3./GUI/KaryMl_UI.py, import the image and click Start Kary Framework, the application doesn't respond and freezes. This error appears after exiting:


[ [C:\Users\a\miniconda3\envs\MEF-GAN\KaryML-Framework-master\GUI\KaryML_UI.py] is applying path hack [C:\Users\a\miniconda3\envs\MEF-GAN\KaryML-Framework-master] before import! ]
2023-03-19 17:56:14 LAPTOP-UK1E4FJ5 OpenGL.acceleratesupport[7320] INFO No OpenGL_accelerate module loaded: No module named 'OpenGL_accelerate'
2023-03-19 17:56:15,562 | 					[mainlogger LOGGER START] = C:\Users\a\miniconda3\envs\MEF-GAN\KaryML-Framework-master\GUI\_logs\common.log

2023-03-19 17:56:16,061 | 					[run_steps_2023-03-19_17-56-16 LOGGER START] = C:\Users\a\miniconda3\envs\MEF-GAN\KaryML-Framework-master\GUI\_logs\KaryPy.log

Start processing using cfg:
{'cols': 50,
 'distance': 'Euclidean',
 'epochs': 200000,
 'feature_names': ['Chromosome length', 'Centromeric Index', 'Banding pattern'],
 'features': [1, 2, 4, 8],
 'input': 'C:/Users/a/Desktop/染色体/BiolmLab/originals/original 4.bmp',
 'mdist': False,
 'pairs': 'C:/Users/a/Desktop/染色体/BiolmLab/karyotypes/karyotype 4.bmp',
 'rows': 50,
 'weights': {1: 0.25, 2: 0.25, 3: 0.25, 4: 0.25}}
2023-03-19 17:56:37,655 | Start segmentation process for image: C:/Users/a/Desktop/染色体/BiolmLab/originals/original 4.bmp
2023-03-19 17:56:37 LAPTOP-UK1E4FJ5 run_steps_2023-03-19_17-56-16[7320] INFO Start segmentation process for image: C:/Users/a/Desktop/染色体/BiolmLab/originals/original 4.bmp

Process finished with exit code -805306369 (0xCFFFFFFF)


It got stuck in the chromosome division step.

I don't understand the reason, could you give me some advice?
